docs/cli-tool/analytics.md

```diff
@@ -1,9 +1,14 @@
 ---
 sidebar_position: 2
+description: Genezio Analytics is a built-in CLI tool feature that gathers telemetry and analytics data. Learn to track and analyze your project’s performance
```

Comment:
```suggestion
description: Genezio Analytics is a built-in feature in the genezio CLI tool that gathers telemetry and analytics data. Learn how to enable or disable it to provide crash reports.
```

Reply:
-This documentation page covers how to disable the telemetry genezio gets from users. It's not related to the end user project performance.